pyspecdata/core.py

```diff
@@ -7415,10 +7415,11 @@ def _taxis(self,taxis):
             taxis = linspace(taxis[0],taxis[1],300)
         return taxis
     def eval(self,taxis,set_what = None,set_to = None):
-        r'''after we have fit, evaluate the fit function along the axis taxis
+        r'''after we have fit, eval calculates the fit function along the axis taxis.
         set_what and set_to allow you to forcibly set_what a specific symbol to a
         specific value --> however, this does not affect the class, but only
-        the return value'''
+        the return value. The specific value defines the number of evenly spaced points
+        along the taxis'''
```

Comment:
no, please read code and write out a full docstring

Reply:
went over and included the parameters and returns for the function hopefully accurate and generated some questions I have int eh fitdata class in general that I see have changes

Reply:
code has been updated and I follow up [here](https://github.com/jmfrancklab/pyspecdata/pull/14/files#r554283732)